nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -218,12 +236,27 @@ define([
          * @param {string} tab id of the tab to display
          */
         toggleTab(tab) {
-            this.state.tabState.selected = tab;
+            var widgetNode = this.ui.getElement('cell-container.tab-pane.widget');
+            if (widgetNode.firstChild) {
+                widgetNode.removeChild(widgetNode.firstChild);
+            }
+
             this.controlPanel.setTabState(this.state.tabState);
             if (this.tabWidget !== null) {
                 this.tabWidget.stop();
             }
-            this.tabWidget = new this.tabSet.tabs[tab].widget();
+            let toggleTab = this.tabSet.tabs[tab];
+
+            if (toggleTab.class) {
+                this.tabWidget = new toggleTab.widget();
+            } else {
```

Comment:
Depending on if a widget is created using a class pattern or a factory pattern, you have to call them differently.